figures/scripts/figS2-snv-callers-panels.R

```diff
@@ -109,9 +126,13 @@ for (dataset in c("tcga", "pbta")) {
       # We'll use this to keep track of the original rows/mutations
       tibble::rowid_to_column("index")
 
+    # Export
+    all_caller_df %>%
+      dplyr::select(Tumor_Sample_Barcode, everything()) %>%
+      dplyr::arrange(Tumor_Sample_Barcode) %>%
+      readr::write_csv(figS2def_csv)
   }
 
-
   ## Upset plots -------------------------------------------------------
   print("Preparing data for upset plot")
   detect_mat <- all_caller_df %>%
```

Comment:
This is no longer being used as (only prep for) a detection matrix, so rename to `vaf_mat`?